queries.sql

```diff
@@ -7,4 +7,51 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+-- Transaction 1
+BEGIN;
+UPDATE ANIMALS SET species='unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+-- Transaction 2
+BEGIN;
+UPDATE ANIMALS SET species='digimon' WHERE name like '%mon';
+UPDATE ANIMALS SET species='pokemon' WHERE species is null;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] As part of the project requirements please make sure to  verify that change was made before committing the changes. This means you'll have to add a statement similar to the one at line `23`